eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/io/AccessionReportWriterTest.java

```diff
@@ -121,46 +128,39 @@ public static String getFirstVariantLine(File output) throws IOException {
 
     @Test
     public void writeInsertionWithAccession() throws IOException {
-        SubmittedVariant variant = new SubmittedVariant("accession", TAXONOMY, "project", CONTIG_1, START_1, "",
-                                                        ALTERNATE, CLUSTERED_VARIANT, false, MATCHES_ASSEMBLY);
+        writeIndeWithAccession("", ALTERNATE, CONTEXT_BASE, CONTEXT_BASE + ALTERNATE);
+    }
 
-        AccessionWrapper<ISubmittedVariant, String, Long> accessionWrapper =
-                new AccessionWrapper<ISubmittedVariant, String, Long>(ACCESSION, "1", variant);
+    private void writeIndeWithAccession(String reference, String alternate, String denormalizedReference,
```

Comment:
`writeIndelWithAccessionHelper` (to fix typo and differentiate more from the actual test methods)